### DISCUSSION TITLE
Lunch and Learn?

### DISCUSSION THREAD

**Discussion by heathdorn00** [keycloak/keycloak#28902] [Q&A]
Hi Keycloak community, I'm looking for a Solution Architect or SME to help provide a lunch and learn on Keycloak.  I have a junior team of Dev's who just started using for identity management and could use your expertise for a 45min session.

**Comment by danielFesenmeyer** [comment]
Not sure whether someone here can afford to do this for free. Hence, it could make sense to mention what you are willing to pay for this training. (But not sure whether this is the right place for job offers.)

As an alternative, you might consider the meetup [Keycloak Hour of Code](https://www.meetup.com/keycloak-hour-of-code/)
